### PR TITLE
jdk 11 standalone TCK and Platform TCK build.

### PR DESCRIPTION
**Pull request by gurunrao**
Signed-off-by: gurunandan.rao@oracle.com <gurunandan.rao@oracle.com>
jdk 11 standalone TCK and Platform TCK build for Jakarta EE 10. 

Platform TCK CI build + run: https://ci.eclipse.org/jakartaee-tck/job/guru/job/jakartaee-tck-build-run/job/jdk11-build/24/
Standalone TCK CI build + run: https://ci.eclipse.org/jakartaee-tck/job/guru/job/jakartaee-tck-build-run/job/jdk11-build/23/

### CODE REVIEW

File: install/caj/bin/ts.jte [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (scottmarlow) [inline install/caj/bin/ts.jte:2]: should be 2021

File: install/concurrency/bin/ts.jte [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2013, 2021 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (scottmarlow) [inline install/concurrency/bin/ts.jte:2]: Why is 2013 changed to 2020?

File: install/jaxrs/bin/ts.jte [hunk 3]
```diff
@@ -77,20 +75,18 @@ javatest.timeout.factor=1
...
+##  RI SPECIFIC PROPERTIES LIVE BELOW
 ###################################################################
 ###################################################################
 ###################################################################
 
 ###################################################################
-## When installing Jakarta RESTful Web Services TCK/CI on Windows, 
-## users must install Jakarta Restful Web Services TCK and
-## the CI on the same drive.  Also note that you should never
+## When installing JAX-RS TCK/RI on Windows, users must install JAXRS-TCK and
```
Comment (scottmarlow) [inline install/jaxrs/bin/ts.jte:84]: looks like we changed Jakarta RESTFul Web Services reference back to JAX-RS.  We should keep the Jakarta RESTFul Web Services

File: install/jaxrs/bin/ts.jte [hunk 4]
```diff
@@ -77,20 +75,18 @@ javatest.timeout.factor=1
 ###################################################################
 ###################################################################
 ###################################################################
-##  CI SPECIFIC PROPERTIES LIVE BELOW
+##  RI SPECIFIC PROPERTIES LIVE BELOW
```
Comment (scottmarlow) [inline install/jaxrs/bin/ts.jte:78]: Please change back to CI instead of RI.

File: install/jaxrs/bin/ts.jte [hunk 5]
```diff
@@ -234,7 +219,7 @@ optional.tech.packages.to.ignore=jakarta.xml.bind
 #    4 - deploy and run only
 #
 # @harness.socket.retry.count - denotes the number of time we should
-#          attempt to create a server socket when initializing a test
+#          attempt to create a server socket when intilizing a test
```
Comment (scottmarlow) [inline install/jaxrs/bin/ts.jte:222]: Please keep correct spelling of `initializing`

File: install/jaxrs/bin/ts.jte [hunk 6]
```diff
@@ -262,7 +247,7 @@ deploy.delay.in.minutes=5
 ###############################################################
 ## These properties must be set to tell the Test harness the
 ## class names of your porting class implementations.  By default
-## both property sets below point to Sun CI specific classes.
+## both property sets below point to Sun RI specific classes.
```
Comment (scottmarlow) [inline install/jaxrs/bin/ts.jte:250]: Change RI back to CI

File: install/jaxrs/bin/ts.jte [hunk 7]
```diff
@@ -274,11 +259,11 @@ porting.ts.HttpsURLConnection.class.1=com.sun.ts.lib.implementation.sun.javaee.S
 
 
 #####################################################################
-## The following properties must be set prior to running the Jakarta RESTful Web Services
+## The following properties must be set prior to running the JAX-RS
```
Comment (scottmarlow) [inline install/jaxrs/bin/ts.jte:262]: Use Jakarta RESTful Web Services instead of JAX-RS

File: install/caj/bin/ts.jte (hunk 1 above)
Comment (gurunrao) [reply install/caj/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. copyright go copied in the process. I have reverted copyright now.

File: install/concurrency/bin/ts.jte (hunk 2 above)
Comment (gurunrao) [reply install/concurrency/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. copyright go copied in the process. I have reverted copyright n

File: install/jaxrs/bin/ts.jte (hunk 3 above)
Comment (gurunrao) [reply install/jaxrs/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.

File: install/jaxrs/bin/ts.jte (hunk 4 above)
Comment (gurunrao) [reply install/jaxrs/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.

File: install/jaxrs/bin/ts.jte (hunk 5 above)
Comment (gurunrao) [reply install/jaxrs/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.

File: install/jaxrs/bin/ts.jte (hunk 7 above)
Comment (gurunrao) [reply install/jaxrs/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.

File: install/jaxrs/bin/ts.jte (hunk 6 above)
Comment (gurunrao) [reply install/jaxrs/bin/ts.jte]: For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.

**Review by scottmarlow — APPROVED** [review]

**Comment by scottmarlow** [comment]
Thanks for the change @gurunrao !

**Pull request merged**